instrumentation/afl-compiler-rt.o.c

```diff
@@ -294,6 +294,21 @@ static void __afl_map_shm_fuzz() {
     u8 *map = NULL;
 
 #ifdef USEMMAP
+
+    // Newer afl-fuzz versions will set a shm_fuzz page size env, else fall back
+    size_t shm_fuzz_map_size = SHM_FUZZ_MAP_SIZE_DEFAULT;
```

Comment:
so we only need this when using USEMMAP? so shm_open does not need that? just to be sure.

Reply:
Looks like it, as far as I understand shm_open just opens the full mapping, whatever size it has

Reply:
`shm_open` (on Linux) is capped by the `tmpfs` without `ftruncate`.